### PR TITLE
Fix migration to terraform-plugin-testing causes panic in existing test suite

### PR DESCRIPTION
**Pull request by bendbennett**
Closes: #1166

### CODE REVIEW

File: helper/resource/aliases.go [hunk 1]
```diff
@@ -0,0 +1,109 @@
+package resource
+
+import (
+	"context"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/retry"
```
Comment (austinvalle) [inline helper/resource/aliases.go:7]: Do we need to add the exports from the new `id` package to this set of aliases as well?

image

**Review by bflad — APPROVED** [review]
Overall looks good, just some things to consider. 👍

File: helper/resource/aliases.go (hunk 1 above)
Comment (bflad) [reply helper/resource/aliases.go:7]: +1

Comment (bendbennett) [reply helper/resource/aliases.go:7]: ++1

**Review by bflad — APPROVED** [review]

**Review by austinvalle — APPROVED** [review]
🚀

**Pull request merged**